docs/ManualTests.md

```diff
@@ -124,13 +124,13 @@ Steps:
     m.	“( )" Parenthesis
 
 
-**All Calulators Test: Verify memory functions**
+**All Calculators Test: Verify memory functions**
```

Comment:
-0. I fixed this in #135, and I'd rather that be merged. (I'm a random contributor like you, but this PR is much more complicated and will require iteration.)